db/migrations/20171109145542_eventum_custom_filter_multiselect.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Eventum (Issue Tracking System) package.
+ *
+ * @copyright (c) Eventum Team
+ * @license GNU General Public License, version 2 or later (GPL-2+)
+ *
+ * For the full copyright and license information,
+ * please see the COPYING and AUTHORS files
+ * that were distributed with this source code.
+ */
+
+use Eventum\Db\AbstractMigration;
+
+class EventumCustomFilterMultiselect extends AbstractMigration
+{
+    const COLUMNS = [
+        'cst_iss_pri_id' => 'cst_priorities',
+        'cst_iss_sev_id' => 'cst_severities',
+        'cst_reporter'   => 'cst_reporters',
+        'cst_iss_prc_id' => 'cst_categories',
+        'cst_iss_sta_id' => 'cst_status',
```

Comment:
should be `cst_statuses` then if it contains multiple values like other fields.

Reply:
I always have trouble making up my mind about that word. The grammatically correct plural is 'status', but it sounds weird and might conflict with a singular use. I'll change it.